server/tserver/src/main/java/org/apache/accumulo/tserver/log/DfsLogger.java

```diff
@@ -186,7 +186,7 @@ public void run() {
         } catch (IOException | RuntimeException ex) {
           fail(work, ex, "synching");
         }
-        long duration = System.currentTimeMillis() - start;
+        long duration = System.nanoTime() - start;
         if (duration > slowFlushMillis) {
           log.info("Slow sync cost: {} ms, current pipeline: {}", duration,
```

Comment:
This should probably convert the display value to milliseconds `NANOSECONDS.toMillis(duration)` - using a static import of NANOSECONDS.